core/templates/core/base.html

```diff
@@ -4,12 +4,41 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <script src="https://cdn.tailwindcss.com"></script>
+        {% load static %}
+        <script src="{% static "tailwind_custom_styles.js" %}"></script>
+        <script src="https://unpkg.com/starback@2.1.1/dist/starback.global.js"></script>
         <title>GwardiaHub | {% block title %}{% endblock %}</title>
+
+        <link href="{% static 'fontawesomefree/css/fontawesome.css' %}" rel="stylesheet" type="text/css">
+        <link href="{% static 'fontawesomefree/css/brands.css' %}" rel="stylesheet" type="text/css">
+        <link href="{% static 'fontawesomefree/css/solid.css' %}" rel="stylesheet" type="text/css">
     </head>
-    <body>
+    <body class="bg-main_bg">
         <main>
+            <canvas id="canvas" class="fixed -z-1"></canvas>
```

Comment:
Don't use id like this

Reply:
What do you mean?

Reply:
`starback-bg` as id would be good enough?